db/db_bolt.go

```diff
@@ -344,7 +344,7 @@ func (b *BoltDB) Insert(name []byte, key uint64, value []byte) error {
 		}
 	}
 	if err != nil {
-		err = errors.Wrap(ErrIO, err.Error())
+		return errors.Wrap(ErrIO, err.Error())
```

Comment:
This changes the behavior of the code (the old code makes no sense). cc @CoderZhi @dustinxie

Reply:
yeah, old code is incorrect